### PR TITLE
Document DataStore limitations

### PR DESCRIPTION
**Pull request by jbothma**
Fixes no issue

### Proposed fixes:

- Document the limitations of the DataStore extension so people don't have to discover it after installation of datastore + datapusher

### Features:

- [ ] includes tests covering changes
- [x] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

File: doc/maintaining/datastore.rst [hunk 1]
```diff
@@ -232,6 +232,8 @@ alongside CKAN.
 
 To install this please look at the docs here: http://docs.ckan.org/projects/datapusher
 
+.. note:: The DataPusher only imports the first worksheet of a spreadsheet. It does also
```
Comment (jbothma) [inline doc/maintaining/datastore.rst]: You might consider moving the "also":

"It also does not support duplicate column headers."

Comment (jbothma) [reply doc/maintaining/datastore.rst]: We could additionally clarify that blank column headings implies duplicate column headings, e.g.

> It also does not support duplicate column headers. That includes blank column headings.

**Comment by amercader** [comment]
Thanks @jbothma. I thought that a quick edit would be the quickest and cleanest but I was obviously wrong :)

**Comment by jbothma** [comment]
Thanks for merging :)

**Pull request merged**